templates/plan_template.html

```diff
@@ -0,0 +1,159 @@
+                            <div class="container d-flex align-items-center justify-content-evenly p-0">
+                                <span class="material-icons">
+                                    account_circle
+                                </span>
+                                <span> Profile </span>
+                            </div>
+                        </a>
+                    </li>
+                    <li id="login-button-item" class="d-none">
+                        <a class="dropdown-item p-0" href="/v1/log-in">
+                            <div class="container d-flex align-items-center justify-content-evenly p-0">
+                                <span class="material-icons">
+                                    login
+                                </span>
+                                <span>Login</span>
+                            </div>
+                        </a>
+                    </li>
+                    <li id="logout-button-item">
+                        <a class="dropdown-item p-0" data-bs-toggle="modal" data-bs-target="#logout-confirm" href="#">
+                            <div class="container d-flex align-items-center justify-content-evenly p-0">
+                                <span class=" material-icons">
+                                    logout
+                                </span>
+                                <span>Logout</span>
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+
+        </div>
+    </nav>
+
+    <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
+        Vacation Planner
+    </h1>
+    <div class="container mt-3" style="align-items: center" id="contains-everything">
+        <div class="input-group mb-3">
+            <input class="form-control-sm border-0" type="date" id="datepicker" name="date" value="2022-01-18">
+            <input type="text" class="form-control" placeholder="Location" id="location">
+        </div>
+
+        <div class="row">
+            <div class="col-12">
+                <table class="table table-bordered" id="template">
+                    <thead class="table-light">
+                        <tr>
+                            <th>Category</th>
+                            <th>Start</th>
+                            <th>End</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    </tbody>
+
+                </table>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="btn-group-sm mb-3" role="group">
+                <button type="button" class="btn btn-outline-primary" id="add-row">add</button>
+                <button type="button" class="btn btn-outline-primary" id="remove-row">remove</button>
+                <button type="button" class="btn btn-outline-success" id="submit"> Search</button>
```

Comment:
How about aligning the value of `id` with its name? Maybe we can change the name from "Search" to "search", all lowercase, so it would be consistent with other buttons